### PR TITLE
mainnet-016: Set Granite PV

### PR DESCRIPTION
**Pull request by Inphi**
Task to set the required and recommended protocol versions on mainnet. This is the first of 4 separate batch transaction upgrades to prepare for the upcoming [Granite protocol upgrade](https://gov.optimism.io/t/upgrade-proposal-10-granite-network-upgrade/8733).

## Sequence

The following table outlines the upgrade sequence, including the nonce allocation of each safe.

| ordering | task | FUS | SC | FOS |
| ---- | ----- | ----- | ---- | ---- |
| **>> 016** | Set recommended/required ProtocolVersions | 7 | | |
| 017 | Resign Presigned Pauses |  |  | 97 |
| [018](https://github.com/ethereum-optimism/superchain-ops/pull/311) | Upgrade FP contracts | 8 | 6 | |
| [019](https://github.com/ethereum-optimism/superchain-ops/pull/312) | Re-enable permissionless FPs | | | 96 |
| [020](https://github.com/ethereum-optimism/superchain-ops/pull/309/) | Upgrade `DeputyGuardianModule` | | 7 | |

### CODE REVIEW

**Comment by sebastianst** [comment]
@Inphi Do we already have a schedule for _when_ these batched transactions are executed? Ideally the required version is bumped 2 days before the Granite mainnet activation.

File: tasks/eth/016-granite-protocol-versions/VALIDATION.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935` (`ProtocolVersions`)
+
+We override the ownership of the PV contract (task#011).
```
Comment (maurelian) [inline tasks/eth/016-granite-protocol-versions/VALIDATION.md:16]: This is an outdated reference.

Comment (Inphi) [reply tasks/eth/016-granite-protocol-versions/VALIDATION.md]: leftover copy n paste from previous pv update. Fixed.

**Review by maurelian — COMMENTED** [review]
A few nits, but overall this LGTM

**Review by maurelian — APPROVED** [review]

**Pull request merged**